charts/karpenter-crd/templates/karpenter.azure.com_aksnodeclasses.yaml

```diff
@@ -249,6 +249,59 @@ spec:
                   - type
                   type: object
                 type: array
+              k8sVersion:
+                description: |-
+                  K8sVersion contains the current k8s version which should be
```

Comment:
Fix in these docs too.

Reply:
You just meaning the description? Not contained in any docs atm.

Reply:
Updated description.